root/components/StaticRelationshipsDisplay.js

```diff
@@ -50,11 +50,18 @@ const StaticRelationshipsDisplay = ({
 
         phraseRows.push(
           <React.Fragment key={targetGroup.key}>
-            {groupSize > 1 && targetGroup.linkOrder ? (
```

Comment:
Also, I removed the `groupSize > 1` from here so that the link order is shown even if there's only one target entity. I see no reason to hide that data from the user in case they need to fix it.